src/OceanSeaIceModels/CrossRealmFluxes/roughness_lengths.jl

```diff
@@ -0,0 +1,132 @@
+                               maximum_roughness_length = 1.6e-4) # Values from COARE3.6
+
+    return ScalarRoughnessLength(air_kinematic_viscosity,
+                                        reynolds_number_scaling_function,
+                                        convert(FT, maximum_roughness_length))
+end
+
+"""
+    MomentumRoughnessLength([FT=Float64];
+                            air_kinematic_viscosity = temperature_dependent_viscosity,
+                            reynolds_number_scaling_function = empirical_scaling_function,
+                            maximum_roughness_length = 1.6e-4)
+
+Constructs a `MomentumRoughnessLength` object that represents the momentum roughness length that
+regulates the exchange of momentum, heat, and water vapor between the ocean and the atmosphere.
+
+Keyword Arguments
+==================
+
+- `gravitational_acceleration::Float`: The gravitational acceleration. Default is `default_gravitational_acceleration`.
+- `maximum_roughness_length::Float`: The maximum roughness length. Default is `1.0`.
+- `air_kinematic_viscosity::Float`: The air kinematic viscosity. Default is `temperature_dependent_viscosity`.
+- `_wave_parameter::Float`: The wave parameter. Default is `0.011`.
+- `laminar_parameter::Float`: The laminar parameter. Default is `0.11`.
+"""
+function MomentumRoughnessLength(FT=Float64;
+                                 gravitational_acceleration = default_gravitational_acceleration,
+                                 maximum_roughness_length = 1.0, # An estimate?
+                                 air_kinematic_viscosity = temperature_dependent_viscosity,
+                                 _wave_parameter = 0.011,
+                                 laminar_parameter = 0.11)
+
+    return MomentumRoughnessLength(convert(FT, gravitational_acceleration),
+                                          air_kinematic_viscosity,
+                                          convert(FT, _wave_parameter),
+                                          convert(FT, laminar_parameter),
+                                          convert(FT, maximum_roughness_length))
+end
+
+function default_roughness_lengths(FT=Float64)
+    momentum    = MomentumRoughnessLength(FT)
+    temperature = ScalarRoughnessLength(FT)
+    water_vapor = ScalarRoughnessLength(FT)
+    return SimilarityScales(momentum, temperature, water_vapor)
+end
+
+# Empirical fit of the scalar roughness length with roughness Reynolds number `R★ = u★ ℓu / ν`
+# Edson et al. (2013), equation (28)
+@inline empirical_scaling_function(R★ :: FT, args...) where FT = 
+        ifelse(R★ == 0, FT(0), convert(FT, 5.85e-5 / R★ ^ 0.72))
+
+# Temeprature-dependent viscosity law: assumes that θ comes in Kelvin
+@inline function temperature_dependent_viscosity(θ :: FT) where FT 
+    T = convert(FT, θ - celsius_to_kelvin)
+    ν = convert(FT, 1.326e-5 * (1 + 6.542e-3 * T + 8.301e-6 * T^2 - 4.84e-9 * T^3))
```

Comment:
this is hard coded!

Reply:
Make `TemperatureDependentViscosity` a type and put these parameters in. Convert the parameters when you make the type